### PR TITLE
RHIDP-3709: how admins can track RHDH usage by users

### PR DESCRIPTION
**Pull request by hmanwani-rh**
License tracking: How admins can track RHDH usage by users

**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):**
1.3 feature

**Issue:**
[RHIDP-3709](https://issues.redhat.com/browse/RHIDP-3709)

**Link to docs preview:**
Find the link within the PR

**Reviews:**
- [x] SME: @PatAKnight 
- [x] QE: @Omar-AlJaljuli 
- [x] Docs review: @Gerry-Forde 
- [ ] Additional review: @mention assignee (by writer)

**Additional information:**

### CODE REVIEW

**Review by PatAKnight — APPROVED** [review]
/lgtm

**Review by Omar-AlJaljuli — APPROVED** [review]
/lgtm

File: modules/admin/con-user-stats-rhdh.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[id='con-user-stats-rhdh_{context}']
+= User statistics in {product}
+
+In {product}, the `licensed-users-info-backend` plugin provides statistical information about the logged-in users using the Web UI or REST API endpoints.
+
+The `licensed-users-info-backend` plugin enables administrators to monitor the number of active users on {product-short}. Using this feature, organizations can compare thier actual usage with the number of licenses they have purchased. Additionally, you can share the user metrics with {company-name} for transparency and accurate licensing.
```
Comment (Gerry-Forde) [inline modules/admin/con-user-stats-rhdh.adoc:6]:
```suggestion
The `licensed-users-info-backend` plugin enables administrators to monitor the number of active users on {product-short}. Using this feature, organizations can compare their actual usage with the number of licenses they have purchased. Additionally, you can share the user metrics with {company-name} for transparency and accurate licensing. 
```

File: modules/admin/con-user-stats-rhdh.adoc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[id='con-user-stats-rhdh_{context}']
+= User statistics in {product}
+
+In {product}, the `licensed-users-info-backend` plugin provides statistical information about the logged-in users using the Web UI or REST API endpoints.
+
+The `licensed-users-info-backend` plugin enables administrators to monitor the number of active users on {product-short}. Using this feature, organizations can compare thier actual usage with the number of licenses they have purchased. Additionally, you can share the user metrics with {company-name} for transparency and accurate licensing. 
+
+The licensed-users-info-backend plugin is enabled by default. This plugin enables a *Download User List* link at the bottom of the *Administration -> RBAC* tab.
```
Comment (Gerry-Forde) [inline modules/admin/con-user-stats-rhdh.adoc:8]:
```suggestion
The `licensed-users-info-backend plugin` is enabled by default. This plugin enables a *Download User List* link at the bottom of the *Administration -> RBAC* tab.
```

File: modules/admin/ref-rbac-rest-api-endpoints.adoc [hunk 3]
```diff
@@ -975,3 +975,80 @@ Deletes a conditional policy for the specified ID.
 ----
 --
 
+== User statistics
+
+The `licensed-users-info-backend` plugin exposes various REST API endpoints to retrieve data related to logged-in users.
+
+There is no additional configuration is required for the `licensed-users-info-backend` plugin. If the RBAC backend plugin is enabled, then an administrator role must be assigned to access the endpoints, as the endpoints are protected by the `policy.entity.read` permission.
```
Comment (Gerry-Forde) [inline modules/admin/ref-rbac-rest-api-endpoints.adoc:982]:
```suggestion
No additional configuration is required for the `licensed-users-info-backend` plugin. If the RBAC backend plugin is enabled, then an administrator role must be assigned to access the endpoints, as the endpoints are protected by the `policy.entity.read` permission.
```

File: modules/admin/ref-rbac-rest-api-endpoints.adoc [hunk 4]
```diff
@@ -975,3 +975,80 @@ Deletes a conditional policy for the specified ID.
...
+    "email": "dev@redhat.com"
+  }
+]
+----
+--
+
+[GET] /users::
++
+--
+Returns the list of logged-in users in CSV format.
```
Comment (Gerry-Forde) [inline modules/admin/ref-rbac-rest-api-endpoints.adoc:1036]:
```suggestion
Returns a list of logged-in users in CSV format.
```

**Review by Gerry-Forde — COMMENTED** [review]
Content looks good to me except for a few minor typos - please see comments.

File: modules/admin/con-user-stats-rhdh.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply modules/admin/con-user-stats-rhdh.adoc:6]: Done

File: modules/admin/con-user-stats-rhdh.adoc (hunk 2 above)
Comment (hmanwani-rh) [reply modules/admin/con-user-stats-rhdh.adoc:8]: Done

File: modules/admin/ref-rbac-rest-api-endpoints.adoc (hunk 3 above)
Comment (hmanwani-rh) [reply modules/admin/ref-rbac-rest-api-endpoints.adoc:982]: Done

File: modules/admin/ref-rbac-rest-api-endpoints.adoc (hunk 4 above)
Comment (hmanwani-rh) [reply modules/admin/ref-rbac-rest-api-endpoints.adoc:1036]: Done

**Review by Gerry-Forde — APPROVED** [review]
/lgtm

**Pull request merged**